README.md

```diff
@@ -77,14 +67,9 @@ data is stored in local storage. This data is retrieved from local storage anyti
 Clone this repository to your desired folder:
 sh
   cd my-folder
- git clone https://github.com/ernestmusong/ernestmusong.github.io.git
+ git clone https://github.com/ernestmusong/Awesome-Books.git
 
 ### Install
-
-Install this project with:
-sh
-  cd ernestmusong.github.io
-  npm install
 ### Usage
 
 ### Run tests
```

Comment:
- [ ] You have these unnecessary sub-sections in your readme file which are making your readme file not professional :+1: Kindly remove Install, Usage, Run tests sub-sections or fill them with commands properly :+1: to make your readme descriptive and professional :+1: